Выпонены задания для homework-17

Drenjy opened the pull request:
Выполнение задания №17

Commit:
Выпонены задания для homework-17

serjs commented on reddit-microservices/.gitignore:
```diff
@@ -3,3 +3,5 @@ prometheus/
 ## python specific
 *.pyc
+
+docker-compose.yml
```
Привет! Убери docker-compose.yml из .gitignore и добавь docker-compose.yml в PR

serjs reviewed (changes requested):
Привет!

Убери compose из ignore - я отписал в строке где нужно внести изменение

И переименуй reddit-microservices/docker-compose.env.example в docker-compose.yml - т.к. это не env файл а сам compose у тебя

Commit:
исправил замечания: "Убери compose из ignore - я отписал в строке где нужно внести изменение

И переименуй reddit-microservices/docker-compose.env.example в docker-compose.yml - т.к. это не env файл а сам compose у тебя"

serjs commented:
Можно закрывать 👍

serjs commented:
Случайно закрыл сам после предыдущего коммента, оставил в апруве и все еще можно мержить и закрывать :)
